src/bin/club-epics.ts

```diff
@@ -23,14 +24,24 @@ const main = async () => {
     const epics = await client.listEpics();
     spin.stop(true);
     const textMatch = new RegExp(program.title, 'i');
-    epics.filter((epic: Epic) => !!`${epic.name} ${epic.name}`.match(textMatch)).map(printItem);
+    const milestoneMatch = new RegExp(program.milestone, 'i');
+    epics
+        .filter((epic: Epic) => {
+            return (
+                !!`${epic.name} ${epic.name}`.match(textMatch) &&
+                !!(program.milestone && `${epic.milestone_id}`.match(milestoneMatch))
```

Comment:
I think we will want to match the `milestone_id` exactly, no? Otherwise, with the regex matching, we could pass milestone ID `1` and get back epics with `milestone_id` `10` and `11` etc.

Reply:
Quite right. Have fixed it. 👍